### PR TITLE
Correct phrasing around dropout rate

### LINKED ISSUE #410: Dropout layer in keras
Currently, in unit 4, the value provided to the dropout function is said to be the amount of neurons that are not dropped. However, the documentation of Keras indicates the opposite that the value provided inside dropout is the rate at which neurons are dropped. See [docs](https://keras.io/api/layers/regularization_layers/dropout/). This should be changed.

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #410

### CODE REVIEW

File: episodes/4-advanced-layer-types.Rmd [hunk 1]
```diff
@@ -526,17 +526,18 @@ Note that the training loss continues to decrease, while the validation loss sta
 
 Techniques to avoid overfitting, or to improve model generalization, are termed **regularization techniques**.
 One of the most versatile regularization technique is **dropout** ([Srivastava et al., 2014](https://jmlr.org/papers/v15/srivastava14a.html)).
-Dropout essentially means that during each training cycle a random fraction of the dense layer nodes are turned off. This is described with the dropout rate between 0 and 1 which determines the fraction of nodes to silence at a time.
+Dropout essentially means that during each training cycle a random fraction of the dense layer nodes are turned off.
```
Comment (svenvanderburg) [inline episodes/4-advanced-layer-types.Rmd:529]: Here I just added a new line, this sentence is actually correct.

File: episodes/4-advanced-layer-types.Rmd [hunk 2]
```diff
@@ -526,17 +526,18 @@ Note that the training loss continues to decrease, while the validation loss sta
...
-As a result, it becomes much harder for a network to memorize particular features. At first this might appear a quiet drastic approach which affects the network architecture strongly.
+As a result, it becomes much harder for a network to memorize particular features. At first this might appear a quite drastic approach which affects the network architecture strongly.
 In practice, however, dropout is computationally a very elegant solution which does not affect training speed. And it frequently works very well.
 
 **Important to note:** Dropout layers will only randomly silence nodes during training! During a predictions step, all nodes remain active (dropout is off). During training, the sample of nodes that are silenced are different for each training instance, to give all nodes a chance to observe enough training data to learn its weights.
 
-Let us add one dropout layer towards the end of the network, that randomly drops 20% of the input units.
+Let us add one dropout layer towards the end of the network, that randomly drops 80% of the nodes.
```
Comment (svenvanderburg) [inline episodes/4-advanced-layer-types.Rmd:540]: This was incorrect. I rephrased input units to nodes. Even though in fact a dropout layer sets its **inputs** to 0, I think for this audience it is better to speak of nodes in the dense layer that are silenced.

File: episodes/4-advanced-layer-types.Rmd (hunk 1 above)
Comment (psteinb) [reply episodes/4-advanced-layer-types.Rmd:529]: As an alternative:
```suggestion
Dropout means that during each training cycle (one forward pass of the data through the model) a random fraction of neurons in a dense layer are turned off.
```

File: episodes/4-advanced-layer-types.Rmd (hunk 2 above)
Comment (psteinb) [reply episodes/4-advanced-layer-types.Rmd:540]: We just ran into this again during a course. This needs to get merged ASAP.

**Review by psteinb — APPROVED** [review]
Approving this as the suggested changes look good to me and the remarks I made can be considered optional.

**Comment by svenvanderburg** [comment]
Thanks for reviewing @psteinb . Annoying that you ran into this again, but it will be the last time! 🥳

**Comment by psteinb** [comment]
Thanks for the merge.
